ports/espressif/common-hal/alarm/pin/PinAlarm.c

```diff
@@ -231,6 +232,7 @@ void alarm_pin_pinalarm_set_alarms(bool deep_sleep, size_t n_alarms, const mp_ob
             if (esp_sleep_enable_ext1_wakeup(1ull << low_pins[1], ESP_EXT1_WAKEUP_ALL_LOW) != ESP_OK) {
                 mp_raise_ValueError(translate("Can only alarm on RTC IO from deep sleep."));
             }
+            esp_sleep_pd_config(ESP_PD_DOMAIN_RTC_PERIPH, ESP_PD_OPTION_ON);
```

Comment:
does this need to be done in the "low_count > 0" (i.e., == 1) case? I don't know what the call does.

.. apparently, no, this is specific to ESP_EXT1_WAKEUP. https://docs.espressif.com/projects/esp-idf/en/latest/esp32s2/api-reference/system/sleep_modes.html#external-wakeup-ext1 assuming all esp32/s2/s3 are identical. I just checked the one page.

Reply:
RIght, I checked that. I'm not sure this is necessary, because there is some "automatic" setting of these power domains based on what is going on elsewhere, but I think it's belt-and-suspenders.